crypto/README.md

```diff
@@ -19,10 +19,13 @@ If you wish to only import the Flow cryptography package to your Go project, ple
 ```
 go get github.com/onflow/flow-go/crypto
 ```
+
+This isn't enough to run the package code as it relies on an extrnal C library ([Relic](https://github.com/relic-toolkit/relic)) for lower level mathematical operations. An extra step is required to compile the external dependency locally. 
+
 - Install [CMake](https://cmake.org/install/), which is used for building the package.
-- From the package directory in `$GOPATH/pkg/mod/github.com/onflow/flow-go/crypto@<version-tag>/`, build the package dependencies. For instance:
+- From the Go package directory in `$GOPATH/pkg/mod/github.com/onflow/flow-go/crypto@<version-tag>/`, build the package dependencies. For instance:
 ```
-cd $GOPATH/pkg/mod/github.com/onflow/flow-go/crypto@v0.12.0/
+cd $GOPATH/pkg/mod/github.com/onflow/flow-go/crypto@v0.18.0/
 go generate
```

Comment:
We should probably document the bash requirement as well (unless and until we replace the build-dependency script with a go program).

Reply:
Do you mean requirements as in the version?